clang/test/CodeGenSYCL/intel-fpga-ivdep-global.cpp

```diff
@@ -75,6 +75,40 @@ void ivdep_conflicting_safelen() {
   }
 }
 
+// Global ivdep w/ safelen value 1 is specified - annotate all GEPs
+//
+// CHECK: define {{.*}}spir_func void @_Z{{[0-9]+}}ivdep_safelen_onev()
+void ivdep_safelen_one() {
+  // CHECK: %[[ARRAY_A:[0-9a-z]+]] = alloca [10 x i32]
+  int a[10];
+  // CHECK: %[[ARRAY_B:[0-9a-z]+]] = alloca [10 x i32]
+  int b[10];
+  [[intel::ivdep(1)]] for (int i = 0; i != 10; ++i) {
```

Comment:
Should the 1 and 0 values STILL be annotating the GEP?

Reply:
I have fixed this problem. 1 and 0 will have no affect, so they should not annotate GEP.